tests/unit_tests/replace_head_test.py

```diff
@@ -37,10 +46,6 @@ def test_dekr_replace_head(self):
         self.assertEqual(heatmap.size(1), 20 + 1)
         self.assertEqual(offsets.size(1), 20 * 2)
 
-    def tearDown(self) -> None:
-        if os.path.exists("~/.cache/torch/hub/"):
-            shutil.rmtree("~/.cache/torch/hub/")
-
```

Comment:
Any reason to remove this ?

Reply:
Yes, as stated in the PR description above: 
> Removed removal of checkpoints cache dir since other tests may reuse the data and not re-download them (saving $$$ on S3 transfer)

:)

Reply:
Haha indeed, sorry 😅